polaris.shopify.com/next.config.js

```diff
@@ -134,6 +134,11 @@ const nextConfig = {
         destination: '/patterns-legacy/text-fields',
         permanent: true,
       },
+      {
+        source: '/patterns-legacy',
+        destination: '/patterns#legacy',
```

Comment:
Just to catch any URLs that've been shared since we made the change; Don't want to leave any 404s in our wake.